schema-guide.md

```diff
@@ -634,14 +756,22 @@ See [`definitions.fax`](#definitionsfax).
 
 The entity's fax number.
 
+Usage example:
+
+```yaml
+authors:
+  - name: The Research Software Project
+    fax: +012-3456-7890
```

Comment:
```suggestion
    fax: +12-3456-7890
```

Just an OCD alert, the `+` replaces the intl `00` "Verkehrsausscheidungsziffer" (there isn't even an English Wikipedia page for telephone country codes 😱, the related Dutch one calls this simply "[Landnummer](https://nl.wikipedia.org/wiki/Landnummer)" I think), and there are countries without the `0` prefix, so <wince> can we not use this here?